Android/BacktraceANRWatchdog.java

```diff
@@ -62,11 +62,11 @@
     /**
      * Initialize new instance of BacktraceANRWatchdog with default timeout
      */
-    public BacktraceANRWatchdog(String gameObjectName, String methodName) {
+    public BacktraceANRWatchdog(String gameObjectName, String methodName, int anrTimeout) {
         Log.d(LOG_TAG, "Initializing ANR watchdog");
         this.methodName = methodName;
         this.gameObjectName = gameObjectName;
-        this.timeout = DEFAULT_ANR_TIMEOUT;
```

Comment:
Is this variable `DEFAULT_ANR_TIMEOUT` still used somewhere or can we delete it now? If it's still used, should we unify with `DefaultAnrWatchdogTimeout`?

Reply:
good point. thanks removed!